app/api/v2/models/redflag_models.py

```diff
@@ -101,30 +87,31 @@ def edit_redflags(self, incidents_id, createdBy):
         dbconn.commit()
 
     def get_incident_by_id(self, id):
-        # Todo
-        # Add other things to update
+        """ Get redflag or interevention details by id"""
         dbconn = self.db
         curr = dbconn.cursor()
         curr.execute(f"SELECT * FROM incidents WHERE incidents_id = {id};")
         incident = curr.fetchall()
         return incident
 
     def update_location(self, location, incidents_id):
+        """ Query to update user location details """
         dbconn = self.db
-        curr=dbconn.cursor()
+        curr = dbconn.cursor()
```

Comment:
avoid using variable names that have no meaningful description

Reply:
Thank you, I will resolve that.